src/main/java/org/folio/linked/data/configuration/ValidatorsConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.linked.data.configuration;
+
+import static org.folio.linked.data.util.Constants.Validators.VALIDATOR_CHAIN_RESOURCE_SAVE;
+import static org.folio.linked.data.util.Constants.Validators.VALIDATOR_CHAIN_RESOURCE_UPDATE;
+
+import org.folio.linked.data.domain.dto.ResourceRequestDto;
+import org.folio.linked.data.service.validation.LdValidator;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ValidatorsConfiguration {
+
+  @Bean(VALIDATOR_CHAIN_RESOURCE_SAVE)
+  public LdValidator<ResourceRequestDto> resourceSaveValidatorsChain(LdValidator<ResourceRequestDto> lccnValidator) {
+    // in future task we can filter out disabled validators here
```

Comment:
One scenario to think about: Let us say, during the application startup the APIs for checking the enable/disable flag is not available. In that case how will the application behave?

Reply:
agree with PK, also we can automatically build a chain of validators by passing List<LdValidator<T>> validators and combine them using Stream#reduce()

Reply:
Regarding the scenario that PK mentioned i think it's a question to PO. If settings are unavailable, should we tell a user that we prevent resource creation/update or we can disable validations for some time? How other modules handle such case?
From technical perspective both cases are fine for us, we can use @Retryable, @Recover annotations and add some logic around them to implements both scenarios.
@pkjacob

Reply:
@pkjacob we can implement any behavior I believe but first we have to agree on the requirements that should be reflected in the ticket in scope of which we are going to implement the configuration part

Reply:
Configuration is out of scope of this ticket, right?
For future: It should be request-scoped I believe, otherwise we can't guarantee that settings are current.